web/src/lib/components/shared-components/search-bar/search-filter-box.svelte

```diff
@@ -4,12 +4,21 @@
   import Icon from '$lib/components/elements/icon.svelte';
   import { getPeopleThumbnailUrl } from '$lib/utils';
   import { handleError } from '$lib/utils/handle-error';
-  import { SearchSuggestionType, type PersonResponseDto } from '@immich/sdk';
+  import {
+    AssetTypeEnum,
+    SearchSuggestionType,
+    type PersonResponseDto,
+    type SmartSearchDto,
+    type MetadataSearchDto,
+  } from '@immich/sdk';
   import { getAllPeople, getSearchSuggestions } from '@immich/sdk';
   import { mdiArrowRight, mdiClose } from '@mdi/js';
-  import { onMount } from 'svelte';
+  import { createEventDispatcher, onMount } from 'svelte';
   import { fly } from 'svelte/transition';
   import Combobox, { type ComboBoxOption } from '../combobox.svelte';
+  import { DateTime } from 'luxon';
+  import _ from 'lodash-es';
```

Comment:
We shouldn't import the entire lodash library but only the methods we need. This reduces package size.